src/cs/usfca/edu/histfavcheckout/filters/AnalyticsFilter.java

```diff
@@ -41,11 +41,10 @@ public void run() {
 				try {
 					boolean success = APIClient.sendEDR(edrRequest);
 					if(success) {
-						//System.out.println("EDR Event responded with non 200 responseCode: " + responseCode + " response!");
-						// LOG here EDR responded non 200 status and log what status was returned. 
+						System.out.println("EDR Event responded with non 200 responseCode: " + response + " response!");
```

Comment:
Why would you log `non 200` if EDR returns success?

the idea of comment here is to say `!success` and then log that EDR responded with error (or non 200 status) response and also mention the response code that we received.